### PR TITLE
ci:hook fps on travis

### PR DESCRIPTION
**Pull request by oakbani**
## Summary
This PR hooks Fullstack Prod Suite build on Travis.

## Test plan
Added FPS Check which should be passed

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -72,6 +74,15 @@ jobs:
...
         - CLIENT=browser $HOME/travisci-tools/trigger-script-with-status-update.sh
       after_success: travis_terminate 0
+
+    - <<: *integrationtest
+      stage: 'Production tests'
+      env:
+        SDK=javascript
+        SDK_BRANCH=$TRAVIS_PULL_REQUEST_BRANCH
+        FULLSTACK_TEST_REPO=ProdTesting
+      script: $HOME/travisci-tools/trigger-script-with-status-update.sh
```
Comment (msohailhussain) [inline .travis.yml:84]: add for client and browser both

Comment (msohailhussain) [reply .travis.yml:84]: WE don't have setup for browser, that's why not added. Checked with Owais.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <h3 align="center">
   Optimizely JavaScript SDK
 </h3>
-
+Sohail Hussain
```
Comment (mikeng13) [inline README.md]: remove this

File: packages/optimizely-sdk/lib/index.node.js [hunk 3]
```diff
@@ -21,7 +21,7 @@ import {
   getErrorHandler,
   LogLevel,
 } from '@optimizely/js-sdk-logging';
-
+sohail hussain
```
Comment (mikeng13) [inline packages/optimizely-sdk/lib/index.node.js]: remove this

File: .travis.yml [hunk 4]
```diff
@@ -27,7 +27,7 @@ stages:
   - name: 'Lint markdown files'
   - 'Lint'
   - 'Integration tests'
-  - 'Production tests'
+  - 'Full stack production tests'
```
Comment (mikeng13) [inline .travis.yml:30]: We should make it explicit that this is for Node: 

`Full stack production tests (Node)`

File: README.md (hunk 2 above)
Comment (msohailhussain) [reply README.md]: Don't know why it's coming up. I reverted 20 hours ago. :) 
https://github.com/optimizely/javascript-sdk/pull/518/commits/4f9c9c7872d2ada69099f45dea2c39d796a0565b

File: .travis.yml (hunk 4 above)
Comment (msohailhussain) [reply .travis.yml:30]: We use same stage, that's why not added `Node`. But status will come -up with node.

**Pull request merged**